website2/src/views/cleanairforum/events/SingleEvent.tsx

```
@@ -198,184 +206,186 @@ const SingleEvent: React.FC<any> = ({ id }) => {
-          </div>
-          <div
-            className="prose lg:prose-xl"
-            dangerouslySetInnerHTML={{
-              __html: eventDetailsHtml || 'Event details coming soon.',
-            }}
-          ></div>
-        </section>
-      )}
+            <div
+              className="prose lg:prose-xl"
+              dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)